RecoTracker/MkFit/plugins/HitSelectionWindowsPhase1.h

```diff
@@ -0,0 +1,5868 @@
+#ifndef RecoTracker_MkFit_plugins_HitSelectionWindowsPhase1_h
+#define RecoTracker_MkFit_plugins_HitSelectionWindowsPhase1_h
+
+namespace mkfit {
+
+  struct HitSelectionWindowsPhase1 {
+  public:
+    float m_dp_params[9][72][3];
```

Comment:
could you please add a comment for helping in understanding these "magic" numbers ?

Reply:
This file is now removed (information was duplicate wrt. the JSON files), JSON file fields are now documented in the README.